x-pack/plugins/triggers_actions_ui/public/application/context/alerts_context.tsx

```diff
@@ -29,6 +29,7 @@ export interface AlertsContextValue<MetaData = Record<string, any>> {
   capabilities: ApplicationStart['capabilities'];
   dataFieldsFormats?: DataPublicPluginSetup['fieldFormats'];
   metadata?: MetaData;
+  data: DataPublicPluginStart;
```

Comment:
This new context parameter definitely should be optional, because it used only by single/few alert types. To avoid the braking of the alerts context usage by the other plugins.

Reply:
In addition to @YulNaumenko 's point, can we give it a more meaningful name? `data` could mean lots of things.
Please make it explicitly something like `dataPlugin` or something like that.

Reply:
This was actually old code and has been removed. In other areas of the code where I do pass data plugin functionality I've given it more meaningful names